README.md

```diff
@@ -1,23 +1,27 @@
 Formality
 =========
 
-Formality is a proof and programming language, as well as a standard format for
-algorithms and proofs. It exists to fill a hole in the current market: there
-aren't many programming languages featuring theorem proving that are 1.
-portable, 2. efficient, 3. accessible.
-
-**Portability** makes it available everywhere, like a JSON of code. We
-accomplish that by bootstrapping: we design a minimal core that can be quickly
-implemented anywhere, and build the rest of the language on top of it.
-
-**Efficiency** is required if we want the language to be used for production
-software. We accomplish that by compiling Formality to a massively parallel,
+Formality is a dependently-typed programming language that enables developers to
+use formal proofs to make their applications safer, faster and friendlier.
+
+## Design Principles
+
+**Portable:** Available everywhere, like a type-safe JSON for code. Formality
+has a minimal core that bootstraps the language and can be quickly implemented
+anywhere.
```

Comment:
You removed the connection between bootstrapping a minimal core and being available everywhere. One allows the other. That connection is important. In your sentence, it seems like you just talk about two unrelated features.

Reply:
I agree that my change obscures that connection, will rewrite ti to make it clearer